source/configuration_manual/dovecot_ssl_configuration.rst

```diff
@@ -1,4 +1,4 @@
-.. _dovecot_ssl_configuration:
```

Comment:
I don't think this was an intentional change.  Also it complains about "undefined label: dovecot_ssl_configuration"